test/src/rabbit_tests.erl

```diff
@@ -3463,3 +3463,34 @@ disconnect_detected_during_alarm([ACfg]) ->
     [] = ListConnections(),
 
     passed.
+
+
+rabbitmqctl_list_consumers(Config) ->
+    StdOut = rabbit_test_configs:rabbitmqctl(Config, "list_consumers"),
+    [<<"Listing consumers", _/binary>> | ConsumerRows] = re:split(StdOut, <<"\n">>, [trim]),
+    CTags = [ lists:nth(3, re:split(Row, <<"\t">>)) || Row <- ConsumerRows ],
+    CTags.
+
+list_consumers_sanity_check_with() ->
+    start_and_connect_a.
+
+list_consumers_sanity_check([ACfg]) ->
+    Chan = pget(channel, ACfg),
+    #'queue.declare_ok'{} = amqp_channel:call(Chan, #'queue.declare'{queue = <<"abc">>}),
```

Comment:
It's not clear how this queue is cleaned up: do we rely on it being exclusive? I'd recommend using a server-generated name either way.